todo_app/README.md

```diff
@@ -76,3 +82,21 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+## Running the Tests
+### Pytest Tests
+Current pytest test cases cover a number of scenarios each with different numbers of items in each list and asserting ...
+ -  the expected number of items in the various lists
+  - whether the app should show all **Done** items in one list or break Done items into **Done (done today)** and **Done Before Today**.  
+
+Presently running the tests pass as expected when run from Test Explorer but not from commandline (unexpected assertion errors happen, to be investigated).
```

Comment:
Thanks for documenting this - comments below (on view_model.py) for why I think this is
Once this is fixed, added some clear instructions for how to run them this way (e.g. with `poetry run pytest`) would be good